opentreemap/treemap/search.py

```diff
@@ -56,6 +56,9 @@ class Filter(object):
     def __repr__(self):
         return "(%s, %s)" % (self.filterstr, self.displaystr)
 
+    def __nonzero__(self):
+        return bool(self.filterstr) and bool(self.displaystr)
```

Comment:
I would personally prefer a named `@property` instead of defining truthyness for a filter. We don't rely on type coercion much in this codebase so I can imagine myself scratching my head when `bool(filter)` returned `False` for a non-None object.

This isn't a nice looking, but it is not bad.:

```python
if filter and filter.is_set:
    ...
```